esx_service/cli/vmdkops_admin_test.py

```diff
@@ -901,48 +955,52 @@ def test_tenant_access(self):
             expected_output = self.datastore_name
             self.assertEqual(expected_output, actual_output)
 
-            # add second access privilege for tenant
```

Comment:
We can retain this test check. Only the default_datastore argument could be removed.